django/otto/templates/cost_dashboard.html

```diff
@@ -201,7 +201,7 @@ <h5 class="card-title visually-hidden">{% trans "Chart options" %}</h5>
           {# Bar chart #}
           <div id="chart-container">
             <canvas id="costChart"
-                    width="1526"
+                    width="1530"
```

Comment:
Amazing this was the actual fix. Could you just add a comment under `{# Bar chart #}` above that explains ratios too close to 2:1 will actually cause bugs?